### PR TITLE
solidstart-fix: clean up code

### PR DESCRIPTION
**Pull request by Megio**
Clean up Solid Start kit

Fixes: #1126

### CODE REVIEW

File: starters/solidstart-tanstackquery-tailwind-modules/README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# SolidStart-TanStackQuery-Tailwind-Modules
```
Comment (hdJerry) [inline starters/solidstart-tanstackquery-tailwind-modules/README.md:1]: Any reason for this?

**Review by hdJerry — COMMENTED** [review]
Why do we have to remove tanstack query?

File: starters/solidstart-tanstackquery-tailwind-modules/README.md (hunk 1 above)
Comment (Megio) [reply starters/solidstart-tanstackquery-tailwind-modules/README.md:1]: the ticket says: “SolidStart” is the name of the framework. Let’s change this in the README, index.tsx, api-example.tsx, and root.tsx files. @kenslachtajr please can you confirm that this is the required change?

**Review by vyktoremario — APPROVED** [review]

File: starters/solidstart-tanstackquery-tailwind-modules/src/routes/api-example.tsx [hunk 2]
```diff
@@ -5,7 +5,7 @@ import PageFooter from '~/components/PageFooter';
 export default function ApiExample() {
   return (
     <>
-      <PageHeader>Solid Start Tanstack Query Fetch Data from API</PageHeader>
+      <PageHeader>Solid Start Fetch Data from API</PageHeader>
```
Comment (vyktoremario) [inline starters/solidstart-tanstackquery-tailwind-modules/src/routes/api-example.tsx:8]: I think ken meant `SolidStart` without the space

File: starters/solidstart-tanstackquery-tailwind-modules/src/routes/index.tsx [hunk 3]
```diff
@@ -4,7 +4,7 @@ export default function Home() {
   return (
     <>
       <header class="flex justify-center items-center text-white my-5 mx-auto bg-blue-500  w-full lg:w-[75%] p-4 text-lg ">
-        Solid Start - Tanstack Query - Tailwind CSS - CSS Modules Kit
+        Solid Start
```
Comment (vyktoremario) [inline starters/solidstart-tanstackquery-tailwind-modules/src/routes/index.tsx:7]: Same as above.

I am not sure he meant for you to remove other technology stacks

**Review by vyktoremario — CHANGES_REQUESTED** [review]
Left some comments

File: starters/solidstart-tanstackquery-tailwind-modules/src/routes/api-example.tsx (hunk 2 above)
Comment (Megio) [reply starters/solidstart-tanstackquery-tailwind-modules/src/routes/api-example.tsx:8]: I'm sooooo stupid 😅

Comment (vyktoremario) [reply starters/solidstart-tanstackquery-tailwind-modules/src/routes/api-example.tsx]: Not really.
I actually thought the same thing you did

**Pull request merged**